aten/src/ATen/native/quantized/cpu/qlinear_dynamic.cpp

```diff
@@ -530,12 +531,19 @@ at::Tensor PackedLinearWeightsOnednn::apply_dynamic_impl(
     x_min = t_min.item<float>();
   }
 #endif
-  const int precision = 8;
+
+:#if defined(__aarch64__) && AT_MKLDNN_ACL_ENABLED()
+  // oneDNN+ACL has optimized kernels for s8s8 matmul, so input is signed
+    using input_qtype = int8_t;
```

Comment:
Fix typos/formatting
```suggestion
#if defined(__aarch64__) && AT_MKLDNN_ACL_ENABLED()
  // oneDNN+ACL has optimized kernels for s8s8 matmul, so input is signed
  using input_qtype = int8_t;
```